### PR TITLE
Feature/arci inte test

### PR DESCRIPTION
**Pull request by pl8787**
Add ArcI integrate test #361

### CODE REVIEW

File: notebooks/wikiqa/dssm.ipynb [hunk 1]
```diff
@@ -616,21 +616,21 @@
...
+    "version": 2
    },
    "file_extension": ".py",
    "mimetype": "text/x-python",
    "name": "python",
    "nbconvert_exporter": "python",
-   "pygments_lexer": "ipython3",
-   "version": "3.6.6"
+   "pygments_lexer": "ipython2",
+   "version": "2.7.10"
```
Comment (bwanglzu) [inline notebooks/wikiqa/dssm.ipynb]: do not commit unrelated file.

File: matchzoo/preprocessor/arci_preprocessor.py [hunk 2]
```diff
@@ -42,12 +42,17 @@ class ArcIPreprocessor(engine.BasePreprocessor, preprocessor.SegmentMixin):
 
     """
 
-    def __init__(self, embedding_file: str=''):
+    def __init__(self, fixed_length: list=[32, 32], embedding_file: str=''):
```
Comment (bwanglzu) [inline matchzoo/preprocessor/arci_preprocessor.py]: `embedding_file: str`

File: matchzoo/models/arci_model.py [hunk 3]
```diff
@@ -52,6 +50,27 @@ def _conv_pool_block(self, input: typing.Any, kernel_count: int,
...
         return output
 
+    @property
+    def embedding_mat(self):
+        """Get pretrained embedding for ArcI model."""
+        # Check if provided embedding matrix
+        if self._params['embedding_mat'] is None:
+            self._params['embedding_mat'] = \
+               np.random.uniform(-0.2, 0.2, (self._params['vocab_size'],
+                                             self._params['embedding_dim']))
```
Comment (bwanglzu) [inline matchzoo/models/arci_model.py]: do not create random numbers.
```python
@property
def embedding_mat(self):
    if not self._params['embedding_mat']:
        raise  TypeError('{...} expected, None received')
    return self._params['embedding_mat']
```

Comment (bwanglzu) [reply matchzoo/models/arci_model.py]: when testing, call `setter` manually to ensure it's not None.

File: matchzoo/models/arci_model.py [hunk 4]
```diff
@@ -52,6 +50,27 @@ def _conv_pool_block(self, input: typing.Any, kernel_count: int,
         output = MaxPooling1D(pool_size=pool_size)(output)
         return output
 
+    @property
+    def embedding_mat(self):
+        """Get pretrained embedding for ArcI model."""
+        # Check if provided embedding matrix
+        if self._params['embedding_mat'] is None:
+            self._params['embedding_mat'] = \
+               np.random.uniform(-0.2, 0.2, (self._params['vocab_size'],
```
Comment (faneshion) [inline matchzoo/models/arci_model.py]: will the range `[-0.2, 0.2]` be able to set as parameters?

File: tests/unit_test/preprocess/test_arci_prerpocessor.py [hunk 5]
```diff
@@ -20,7 +20,8 @@ def test_arci_preprocessor_noembed(train_inputs, validation_inputs):
     assert len(rv_train.left) == 1
```
Comment (faneshion) [inline tests/unit_test/preprocess/test_arci_prerpocessor.py]: Why not directly assert the `value` rather than the `len`? e.g., `asset rv_train.left == [...]`.
